docs/_guide/07-tools.md

```diff
@@ -0,0 +1,194 @@
+
+If you're using webpack for your build process, you can also use the webpack dev server.
+
+#### Polymer Dev Server
+
+Install the Polymer CLI:
+
+```bash
+npm i -g polymer-cli
+```
+
+Run the dev server:
+
+```bash
+polymer serve
+```
+
+The Polymer CLI was designed to help develop, test, and build projects using web components, JavaScript modules and other features of the modern web platform. It's not required for lit-html development, but it provides several handy utilities. 
+
+#### Open Web Components Dev Server
+
+The Open Web Components project produces a dev server that handles remapping node-style modules to browser-style modules.
+
+For full installation and usage instructions, see the [open-wc website](https://open-wc.org/developing/owc-dev-server.html). 
+
+## Testing
+
+lit-html doesn't have many special testing requirements. If you already have a testing setup, it should work fine as long as it supports working with JavaScript modules (and node-style module specifiers, if you use them).
+
+Web Component Tester (WCT) is an end-to-end testing environment that supports node-style module specifiers. works with the Mocha testing framework and (optionally) the Chai assertion library. There are two ways to add WCT to your project:
+
+* [web-component-tester](https://www.npmjs.com/package/web-component-tester).  Installing the full WCT package gives you Mocha and Chai, as well as some other add-ons.
+* [wct-mocha](https://www.npmjs.com/package/wct-mocha). Just the WCT client-side library. You'll need to install your own version of Mocha, and any other add-ons you want.
+
+Alternately, you can also use the Karma test runner. The Open Web Components recommendations includes a [Karma setup](https://open-wc.org/testing/testing-karma.html#browser-testing) that resolves module dependencies by bundling with webpack before running tests. 
+
+## Build
+
+Build tools take your code and make it production-ready. Among the things you may need build tools to do:
+
+* Transform ES6 code to ES5 for legacy browsers, including transforming JavaScript modules into other formats.
+* Bundle modules together can improve performance by reducing the number of files that need to be transferred. 
+* Minify JavaScript, HTML, and CSS.
+
+Many build tools can do this for you. Currently we recommend the Polymer CLI or webpack. 
+
+The Polymer CLI includes a set of build tools that can handle lit-html with minimal configuration.
+
+webpack is a powerful build tool with a large ecosystem of plugins. The [Open Web Components](https://open-wc.org/building/#webpack) project provides a default configuration for webpack that works well for lit-html and LitElement.
+
+Other tools such as Rollup can work, too. If you're using another tool or creating your own webpack configuration, see the section on [Build considerations for other tools](#build-consderations).
+
+### Build your project with Polymer CLI
+
```

Comment:
I think it's important to mention here that building & bundling the JavaScript should be left to applications, and the only building that we recommend reusable libraries do is compile down to ES2017. Kate has a nice section on this in the new LitElement publishing docs.